src/store/modules/topology.ts

```diff
@@ -338,14 +338,18 @@ export const topologyStore = defineStore({
       }
       const res = await this.getEndpointTopology(endpointIds);
       if (depth > 1) {
-        const ids = res.nodes.map((item: Node) => item.id).filter((d: string) => !endpointIds.includes(d));
+        const ids = res.nodes
+          .filter((item: Node) => item.isReal)
```

Comment:
A database could be `not real`. I think we should only filter User out.

Reply:
> Yes, I think that is.

Okay, I'll make the changes now